Content.IntegrationTests/Tests/InteractionVerbs/InteractionPrototypesTest.cs

```diff
@@ -0,0 +1,37 @@
+using System.Linq;
+using Content.Client.Guidebook;
+using Content.Server.Verbs;
+using Content.Shared.InteractionVerbs;
+using Content.Shared.Verbs;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests.InteractionVerbs;
```

Comment:
woah new test that's cool

definitely needs another maintainer to look at this part as I have no idea how tests operate